spec/liquid/liquid_renderer_spec.rb

```diff
@@ -163,13 +163,13 @@
       it 'has the fields from one plugin form' do
         form = create :form_with_email_and_name
         create :plugins_fundraiser, page: page, form: form
-        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name']
+        expect(LiquidRenderer.new(page, layout: liquid_layout).personalization_data['outstanding_fields']).to eq ['email', 'name', 'country']
       end
 
       it "checks with the member's liquid data" do
         form = create :form_with_email_and_name
         create :plugins_fundraiser, page: page, form: form
-        member = create :member, name: 'Humphrey Bogart', email: 'psycho@killer.com'
+        member = create :member, name: 'Humphrey Bogart', email: 'psycho@killer.com', country: 'US'
```

Comment:
This is still weird though... if we call `create :form_with_email_and_name`, it shouldn't also have a country field. If `country` is still coming back on `outstanding_fields` it means a) the factory is wrong and does in fact include country, which we should make it not, or b) outstanding fields is messing up and validating against something that isn't the real fields of the form. either way it seems like you've discovered a bug.